reference/api/overview.md

```diff
@@ -84,6 +84,26 @@ Meilisearch currently supports the following formats:
 
 Only the [add documents](/reference/api/documents.md#add-or-replace-documents) and [update documents](/reference/api/documents.md#add-or-update-documents) endpoints accept NDJSON and CSV. For all others, use `Content-Type: application/json`.
 
+### Content encoding
+
+The `Content-Encoding` header indicates that the media type is compressed by a given algorithm. It improves transfer speed and reduces bandwidth consumption by sending and receiving a smaller, compressed payload. The `Accept-Encoding` header indicates the compression algorithm the client understands. Meilisearch supports the following compression methods:
```

Comment:
Since `content-encoding` and `accept-encoding` are so similar, I feel the "instead" helps reinforcing they're different things.

```suggestion
The `Content-Encoding` header indicates the media type is compressed by a given algorithm. Compression improves transfer speed and reduces bandwidth consumption by sending and receiving smaller payloads. The `Accept-Encoding` header,  instead, indicates the compression algorithm the client understands. 

Meilisearch supports the following compression methods:
```